skeleton/model_servers/README.md

```diff
@@ -0,0 +1 @@
+The models in this directory are sourced from https://github.com/containers/ai-lab-recipes/tree/main/model_servers
```

Comment:
how are we generating these files? Is it just a manual dump? Would be better in the long term, if we just automate it and copy it here with a script 🤔

Reply:
Yeah I just manually dumped them to get up and running, I'm not sure if we want to automate the updating/grabbing of new models or just leave it as it for sampling?